play/src/front/Chat/Components/RoomFolder.svelte

```diff
@@ -37,6 +37,16 @@
             class="tw-flex tw-items-center tw-space-x-2 tw-grow tw-m-0 tw-p-0"
             on:click={() => {
                 isOpen = !isOpen;
+
+                if (isOpen) {
+                    const foldersOpened = localUserStore.getFoldersOpened();
+                    foldersOpened.add(id);
+                    localUserStore.setFoldersOpened(foldersOpened);
+                } else {
+                    const foldersOpened = localUserStore.getFoldersOpened();
+                    foldersOpened.delete(id);
+                    localUserStore.setFoldersOpened(foldersOpened);
+                }
```

Comment:
Replace getFoldersOpened and setFoldersOpened by localUserStore.hasFolderOpened localUserStore.setFolderOpened and localUserStore.removeFolderOpened